src/cbmc/Makefile

```diff
@@ -1,6 +1,5 @@
 SRC = all_properties.cpp \
       bmc.cpp \
-      bmc_cover.cpp \
```

Comment:
This has already been moved in the previous commit, hence the prior commit yields an inconsistent state.